rosetta-source/src/main/rosetta/product-template-type.rosetta

```diff
@@ -182,23 +182,23 @@ type InitialMarginCalculation: <" Defines the initial margin calculation applica
 
 type ReturnTerms: <"Specifies the type of return of a performance payout.">
 
-    // commodityReturnTerms CommodityPriceReturnTerms (0..1)
     priceReturnTerms PriceReturnTerms (0..1) <"Return terms based upon the underlier's observed price.">
     dividendReturnTerms DividendReturnTerms (0..1) <"Return terms based upon dividend payments associated to the underlier.">
     varianceReturnTerms VarianceReturnTerms (0..1) <"Return terms based upon the observed variance of the underlier's price.">
     volatilityReturnTerms VolatilityReturnTerms (0..1) <"Return terms based upon the observed volatility of the underlier's price.">
     correlationReturnTerms CorrelationReturnTerms (0..1) <"Return terms based upon the observed correlation between the components of the underlying basket.">
 
-    condition ReturnTermsExists: <"Checks that the return type label matches the actual return terms structure of the product.">
-        if priceReturnTerms -> returnType = ReturnTypeEnum -> Total
-        then (priceReturnTerms, dividendReturnTerms) only exists
-        else if priceReturnTerms -> returnType = ReturnTypeEnum -> Price
-        then priceReturnTerms only exists
-        else priceReturnTerms only exists
-            or dividendReturnTerms only exists
-            or varianceReturnTerms only exists
-            or volatilityReturnTerms only exists
-            or correlationReturnTerms only exists
+  condition ReturnTermsExists: <"Checks that the return type label matches the actual return terms structure of the product.">
```

Comment:
As previously discussed: we should not look to modify the conditions here. Instead, this raises a DSL bug, for which a fix is planned in an upcoming release.

Revert changes.